packages/console/src/notebook/ScriptEditor.tsx

```diff
@@ -168,10 +177,14 @@ class ScriptEditor extends Component<ScriptEditorProps, ScriptEditorState> {
     if (focusOnMount != null && focusOnMount) {
       innerEditor.focus();
     }
+
+    onEditorInitialized(this.editor);
   }
 
   handleEditorWillDestroy(): void {
```

Comment:
Take in the editor param here and pass it through

Reply:
This works fine, I would have passed the value through `handleEditorWillDestroy` and used that though (then no need for `assertNotNull`)